packages/contracts/docs/core/01-how-it-works/01-the-core-contracts/02-permissions.md

```diff
@@ -155,15 +158,14 @@ However, some restrictions apply. For security reasons, aragonOS does not allow
 
 The following functions in the DAO are permissioned:
 
-| Functions                                      | Permission Identifier                   | Description                                                                            |
-| ---------------------------------------------- | --------------------------------------- | -------------------------------------------------------------------------------------- |
-| `execute`                                      | `EXECUTE_PERMISSION_ID`                 | Required to execute arbitrary actions.                                                 |
-| `withdraw`                                     | `WITHDRAW_PERMISSION_ID`                | Required to withdraw assets from the DAO.                                              |
-| `_authorizeUpgrade`                            | `UPGRADE_DAO_PERMISSION_ID`             | Required to upgrade the DAO (via the [UUPS](https://eips.ethereum.org/EIPS/eip-1822)). |
-| `setMetadata`                                  | `SET_METADATA_PERMISSION_ID`            | Required to set the DAO’s metadata.                                                    |
-| `setTrustedForwarder`                          | `SET_TRUSTED_FORWARDER_PERMISSION_ID`   | Required to set the DAO’s trusted forwarder for meta transactions.                     |
-| `setSignatureValidator`                        | `SET_SIGNATURE_VALIDATOR_PERMISSION_ID` | Required to set the DAO’s signature validator contract (see ERC-1271).                 |
-| `grant`, `grantWithCondition`, `revoke`           | `ROOT_PERMISSION_ID`                    | Required to manage permissions of the DAO and associated plugins.                      |
+| Functions                               | Permission Identifier                   | Description                                                                            |
+| --------------------------------------- | --------------------------------------- | -------------------------------------------------------------------------------------- |
```

Comment:
missing `registerStandardCallback`, `setDaoURI`,

Reply:
Noted, I will do it in my other task

Reply:
Yes, we do the rest of the docs separately.
Just focus on the smart contract code so that we don't forget something.